### PR TITLE
Pause Buyback swapping

### PR DESCRIPTION
**Pull request by sparrowDom**
Pause OUSD to OGN Buyback swapping functionality. 

[Snapshot vote](https://vote.originprotocol.com/#/proposal/0x978f6613eb09c9bae26e7596d4d437675ecbf4b469efffb4a3d20d0ebcd4322f)

### CODE REVIEW

File: brownie/addresses.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 COMP_STRAT = '0x9c459eeb3FA179a40329b81C1635525e9A0Ef094'
 CONVEX_STRAT = '0xEA2Ef2e2E5A749D4A66b41Db9aD85a38Aa264cb3' 
 FLIPPER = '0xcecaD69d7D4Ed6D52eFcFA028aF8732F27e08F70'
-BUYBACK = '0x7294cd3c3eb4097b03e1a61eb2ad280d3dd265e6'
```
Comment (DanielVF) [inline brownie/addresses.py:19]: Is this supposed to be in this PR?

File: brownie/scripts/test_stop_buyback.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from world import *
+
+# send a lot of OUSD to buyback
+OUSD_BAGS = '0x8e02247d3ee0e6153495c971ffd45aa131f4d7cb'
+ousd.transfer(buyback, ousd.balanceOf(OUSD_BAGS), {'from': OUSD_BAGS})
+
+# cofirm address is set to 0x0
```
Comment (tomlinton) [inline brownie/scripts/test_stop_buyback.py:7]: Typo

File: contracts/deploy/041_stop_buyback_swaps.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+const { deploymentWithProposal } = require("../utils/deploy");
+module.exports = deploymentWithProposal(
+  { deployName: "041_stop_buyback_swaps", forceDeploy: false },
+  async ({}) => {
+    // Buyback contract
+    const cBuyback = await ethers.getContract("Buyback");
+
+    // Governance proposal
+    return {
+      name: "Deploy all new contracts and migrate all funds",
```
Comment (franckc) [inline contracts/deploy/041_stop_buyback_swaps.js:13]: Update the proposal name to reflect what this deploy actually does.

File: brownie/addresses.py (hunk 1 above)
Comment (sparrowDom) [reply brownie/addresses.py:19]: yes the current buyback address in the brownie script was outdated

File: contracts/deploy/041_stop_buyback_swaps.js (hunk 3 above)
Comment (sparrowDom) [reply contracts/deploy/041_stop_buyback_swaps.js:13]: thanks fixed 👍

**Comment by sparrowDom** [comment]
The [vote has passed](https://vote.originprotocol.com/#/proposal/0x978f6613eb09c9bae26e7596d4d437675ecbf4b469efffb4a3d20d0ebcd4322f) and comments addressed. This should be ok for deploy IMHO

**Comment by sparrowDom** [comment]
@DanielVF could you maybe take over deploying this thing?

**Comment by franckc** [comment]
@sparrowDom Hit me up on discord and I can bring you up to speed on our contract deployment process. So that dvf and tomlinton are not the only ones who are familiar with it...

**Pull request merged**